packagingenv/centos8/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM centos:8
+
+RUN yum -y install gcc-c++ make git libicu-devel rpmdevtools
+
+# Add NodeSource repo
+RUN curl --silent --location https://rpm.nodesource.com/setup_10.x | bash -
+
+# Install development tools
+RUN yum -y install nodejs
```

Comment:
Oh well, `nodejs` on centos8 is installing `python2` as a dependency :disappointed_relieved: 

```
[vagrant@centos8 ~]$ sudo yum install nodejs
Last metadata expiration check: 0:23:01 ago on Fri 17 Jan 2020 05:30:48 PM UTC.
Dependencies resolved.
===============================================================================================================================================
 Package                               Arch                Version                                               Repository               Size
===============================================================================================================================================
Installing:
 nodejs                                x86_64              2:10.18.1-1nodesource                                 nodesource               20 M
Installing dependencies:
 python2                               x86_64              2.7.16-12.module_el8.1.0+219+cf9e6ac9                 AppStream               109 k
 python2-libs                          x86_64              2.7.16-12.module_el8.1.0+219+cf9e6ac9                 AppStream               6.0 M
 python2-pip-wheel                     noarch              9.0.3-14.module_el8.1.0+219+cf9e6ac9                  AppStream               1.2 M
 python2-setuptools-wheel              noarch              39.0.1-11.module_el8.1.0+219+cf9e6ac9                 AppStream               289 k
Installing weak dependencies:
 python2-pip                           noarch              9.0.3-14.module_el8.1.0+219+cf9e6ac9                  AppStream               2.0 M
 python2-setuptools                    noarch              39.0.1-11.module_el8.1.0+219+cf9e6ac9                 AppStream               643 k

Transaction Summary
===============================================================================================================================================
Install  7 Packages

Total download size: 30 M
Installed size: 99 M
Is this ok [y/N]: 
```

https://github.com/nodesource/distributions/issues/845#issuecomment-564328502

Reply:
@armab I found that thread yesterday myself...

Reply:
I am trying `yum install -y nodejs:10`

Reply:
https://github.com/StackStorm/st2chatops/issues/142